src/main/kotlin/io/github/ackeecz/danger/junit/JUnitPlugin.kt

```diff
@@ -96,8 +96,8 @@ internal data class TestCase(
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 internal data class Failure(
-    @field:JacksonXmlProperty val message: String = "",
-    @field:JacksonXmlProperty val type: String = "",
+    @field:JacksonXmlProperty(isAttribute = true) val message: String = "",
```

Comment:
These two lines are the fix for being able to parse failures with no `message` and/or `type`.